kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/i18n/ResourceBundleTranslations.kt

```diff
@@ -120,7 +120,7 @@ public open class ResourceBundleTranslations(
         return result
     }
 
-    override fun translate(key: String, locale: Locale, bundleName: String?, replacements: Array<Any?>): String {
+    private fun getTranslatedString(key: String, locale: Locale, bundleName: String?): String {
```

Comment:
This should not be `private` - mark it `protected` instead. Do not mark anything `private` on open classes.

Reply:
Changed it to protected, and added kdoc string